spec/gcs_spec.rb

```diff
@@ -4,4 +4,16 @@
   it "has a version number" do
     expect(Gcs::VERSION).not_to be nil
   end
+
+  if $credential_available
+    describe "read_partial" do
```

Comment:
I prefer to use `skip` instead of `if`, like this:

```ruby
  describe "read_partial", skip: !$credential_available do
    before do
      ...(snip)...
    end
    it do
      ...(snip)...
    end
  end
```